app/femr/ui/controllers/admin/InventoryController.java

```diff
@@ -208,11 +208,26 @@ public Result ajaxGet() {
 
     }
 
-    public Result ajaxDelete(int medicationID) {
-        ServiceResponse<MedicationItem> inventoryServiceResponse = medicationService.deleteMedication(medicationID);
+    public Result ajaxDelete(int medicationID, int tripId) {
+        ServiceResponse<MedicationItem> inventoryServiceResponse = inventoryService.deleteInventoryMedication(medicationID, tripId);
         if (inventoryServiceResponse.hasErrors()) {
             throw new RuntimeException();
         }
         return ok("true");
     }
+
+
+    /**
+     * Alters medication based on submit.
+     */
+    public Result editPost(int medicationID, int quantity, int tripId) {
```

Comment:
This might be better off being named ajaxEdit to stay in line with ajaxDelete

I would see editGet/editPost being the method names for the comprehensive edit page